src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/services/StudyServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.persistence.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Study;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Taxonomy;
+import uk.ac.ebi.ampt2d.metadata.persistence.repositories.StudyRepository;
+import uk.ac.ebi.ampt2d.metadata.persistence.repositories.TaxonomyRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component("studyService")
```

Comment:
@Service in any case, and you should not need the name If you need the name we need to check the configuration of the application. 

Anyway lately I'm favoring to do not mark them as searchable beans and making the instantiation as @Beans on @Configuration files, check other spring projects of the group.

Reply:
I have moved the @Component annotation to a @bean in the @configuration .